Task_DEV_4/Task_DEV_4/Triangle.cs

```diff
@@ -0,0 +1,56 @@
+﻿using System;
+
+namespace Task_DEV_4
+{
+    class Triangle : Figure
+    {
+        private const string MESSAGE_ERROR_FIRST = "Side cannot be less or equals than 0.";
+        private const string MESSAGE_ERROR_SECOND = "The sum of the two sides " +
```

Comment:
The same as in comments above.

Reply:
smth like TRIANGLE_EXISTANCE_ERROR_MESSAGE

Reply:
Done.